src/test/javascript/spec/component/modeling-editor/modeling-editor.component.spec.ts

```diff
@@ -225,4 +226,47 @@ describe('ModelingEditorComponent', () => {
         const spanText = statusHint.query(By.css('span'))?.nativeElement?.textContent;
         expect(spanText).toBe('Saving...');
     });
+
+    it('should handle explanation input change', () => {
+        fixture.detectChanges();
```

Comment:
Is this change detection needed? As far as I can see nothing happend so far, so there should be no data that has to be binded.

Reply:
Yes you are right, the test still passed without that line so I removed it.